cwms-data-api/src/test/java/cwms/cda/data/dto/MeasurementTest.java

```diff
@@ -258,4 +283,269 @@ void createMeasurement_deserialize() throws Exception {
+                .build();
+
+        SupplementalStreamflowMeasurement supplementalStreamflowMeas2 = new SupplementalStreamflowMeasurement.Builder()
+                .withChannelFlow(320.0)
+                .withOverbankFlow(45.0)
+                .withOverbankMaxDepth(4.5)
+                .withChannelMaxDepth(9.5)
+                .withAvgVelocity(1.8)
+                .withSurfaceVelocity(2.5)
+                .withMaxVelocity(3.5)
+                .withEffectiveFlowArea(220.0)
+                .withCrossSectionalArea(260.0)
+                .withMeanGage(21.0)
+                .withTopWidth(32.0)
+                .withMainChannelArea(160.0)
+                .withOverbankArea(85.0)
+                .build();
+
+        UsgsMeasurement usgsMeas2 = new UsgsMeasurement.Builder()
+                .withRemarks("Post-rain conditions.")
+                .withCurrentRating("Updated Rating")
+                .withControlCondition("Adjusted Condition")
+                .withFlowAdjustment("Flow Adjustment Applied")
+                .withShiftUsed(0.15)
+                .withPercentDifference(4.5)
+                .withDeltaHeight(0.1)
+                .withDeltaTime(15.0)
+                .withAirTemp(18.0)
+                .withWaterTemp(16.0)
+                .build();
+
+        Measurement expectedMeasurement2 = new Measurement.Builder()
+                .withId(cwmsId2)
+                .withHeightUnit("ft")
+                .withFlowUnit("cfs")
+                .withTempUnit("F")
+                .withVelocityUnit("ft/s")
+                .withAreaUnit("sq ft")
+                .withUsed(false)
+                .withAgency("USGS")
+                .withParty("Second Survey Party")
+                .withWmComments("Measurement made after recent rainfall.")
+                .withInstant(Instant.parse("2024-09-17T12:00:00Z"))
+                .withNumber("654321")
+                .withStreamflowMeasurement(sfm2)
+                .withSupplementalStreamflowMeasurement(supplementalStreamflowMeas2)
+                .withUsgsMeasurement(usgsMeas2)
+                .build();
+
+        // Reading and Deserializing the JSON
+        InputStream resource = this.getClass().getResourceAsStream("/cwms/cda/data/dto/measurements.json");
+        assertNotNull(resource);
+        String json = IOUtils.toString(resource, StandardCharsets.UTF_8);
+        ContentType contentType = new ContentType(Formats.JSON);
+        List<Measurement> deserialized = Formats.parseContentList(contentType, json, Measurement.class);
+
+        // Assert both measurements were deserialized correctly
+        assertEquals(2, deserialized.size(), "Expected 2 measurements");
+        DTOMatch.assertMatch(expectedMeasurement1, deserialized.get(0));
+        DTOMatch.assertMatch(expectedMeasurement2, deserialized.get(1));
+    }
+
+    @Test
+    void testRoundTripMultiple()
+    {
+        CwmsId cwmsId1 = new CwmsId.Builder()
+                .withName("Location123")
```

Comment:
Use a real location ID, I didn't notice this till now, but can you make sure the other locations are using

Reply:
Updated to try to use real location IDs... Though i don't think it matters much for the purpose of unit testing the DTOs since they aren't integration tests against real data.